packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js

```diff
@@ -71,6 +71,15 @@ const Decorator = hoc(defaultConfig, (config, Wrapped) => {
 			 */
 			popupComponent: PropTypes.func.isRequired,
 
+			/**
+			 * When `true`, the range of voice control is limited to popup.
+			 *
+			 * @type {Boolean}
```

Comment:
This needs an `@memberof` to doc correctly.  I'll fix it.

Reply:
Second point:  Are there use cases where would we not want to base this off the `spotlightRestrict` setting so it matches with 5-way and pointer mode restrictions?  If `self-only` set the restrict `true` and if not, set to `false`?  Or, would we want to make it nullable and allow it to follow the default above unless specifically set to `false` or `true`?

Reply:
In fact, the policy of voice control should be set separately from spotlightRestrict.
First, when popup is opened, the basic UX policy is to limit the voice control area to pop-in.
For example, if spotlightRestrict is set to self-first or none, the voice control range is limited to pop-up internals.
However, for ContexturePopup, there are scenarios where exception handling is required.
For example, ContexturePopup is used in the Settings menu during video playback in the Photo Video app.
In this case, if we limit the voice control area to pop-up, the Intent(PlayListControl) related to video control will not work.
(The web engine gathers intent information based on the dom node, and if the data-webos-voice-exclusive attribute is present, limits the scope of the queryselector to the subordinate attribute)